src/components/Button/Base.js

```diff
@@ -25,12 +25,16 @@ class Button extends Component {
   };
 
   blurHandler = () => {
-    this.button.current.classList.add("noFocus");
+    if (this.button && this.button.current && this.button.current.classList) {
```

Comment:
Hmm wonder why we didn't use pseudo-selectors here, rather than `classNames` 🤔

Reply:
That's a question.
Just ran into an error of `can not call add() on null` or smth like that.